book_manager.rb

```diff
@@ -0,0 +1,30 @@
+# Class responsible for book management
+class BookManager
+  attr_accessor :books
+
+  def initialize
+    @books = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No books available.'
+    else
+      puts 'List of all books:'
+      @books.each_with_index do |book, index|
+        puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
+      end
+    end
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+
+    new_book = Book.new(title, author)
+    @books << new_book
+    puts 'Book created successfully'
+  end
+end
```

Comment:
- [ ] As pointed out earlier, please kindly ensure that this method is implemented in accordance to Single Responsibility Rule. The principle states that a class should have only one reason to change, meaning it should have only one responsibility or concern. But here, this method is making requests for **input validation + creating a new book**. `All requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.`

### **Note: Kindly refactor `create_teacher` (in person_manager.rb directory), `create_student` (in person_manager.rb directory) and `create_new_rental` (in rental_manager.rb directory) methods in accordance to SRP** 🙏 🙏 

## Commendation: The implementation for `create_person` method is correct. ✔️ ✔️

Reply:
sorry i dont undesrstand, what yo suggest is to remove the create_book / create_student / create_new_rental and add that methods into our app.rb file?

Reply:
@Felipe-Perez-Ferraro I'm sorry I just saw your message, kindly use the "@" tag next time so that I can be notified. 🙏